### PR TITLE
fix(navigation-secondary): realign color-palette with surface tokens

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1. Realigned `color-palette` default to new surface token values

## Testing Instructions

1. View [Deploy Preview,](https://deploy-preview-878--red-hat-design-system.netlify.app/elements/navigation-secondary/demo/) navigation-secondary background should match `--rh-color-surface-lighter` value of #f2f2f2

### CODE REVIEW

**Comment by zeroedin** [comment]
DP is broken due to https://github.com/RedHat-UX/red-hat-design-system/issues/874

File: elements/rh-navigation-secondary/rh-navigation-secondary.ts [hunk 1]
```diff
@@ -106,7 +106,7 @@ export class RhNavigationSecondary extends LitElement {
 
 
   @colorContextProvider()
-  @property({ reflect: true, attribute: 'color-palette' }) colorPalette: NavPalette = 'light';
+  @property({ reflect: true, attribute: 'color-palette' }) colorPalette: NavPalette = 'lighter';
 
   /**
    * If the host color-palette="light", the cta color context should be on="light"
```
Comment (brianferry) [inline elements/rh-navigation-secondary/rh-navigation-secondary.ts:112]:
```suggestion
   * If the host color-palette="lighter", the cta color context should be on="lighter"
```

Comment (brianferry) [reply elements/rh-navigation-secondary/rh-navigation-secondary.ts:112]: Same on line 114

**Review by marionnegp — APPROVED** [review]
👍

**Pull request merged**